libnavigation-ui/src/main/res/values/colors.xml

```diff
@@ -35,7 +35,7 @@
   <color name="mapbox_navigation_view_color_primary_dark">#2D3F53</color>
   <color name="mapbox_navigation_view_color_secondary_dark">#D8D8D8</color>
   <color name="mapbox_navigation_view_color_accent_dark">#FFFFFF</color>
-  <color name="mapbox_navigation_view_color_primary_text_dark">#424242</color>
+  <color name="mapbox_navigation_view_color_primary_text_dark">#FFFFFF</color>
```

Comment:
It says text dark but the color is #FFFFFF which is white. Just wanted to make sure that it is not a typo

Reply:
yes, it's not typo. Actually it's fixing an existing typo issue in the old code base.
you could see that we are not using this color before where we should.